lib/rspec/expectations/minitest_integration.rb

```diff
@@ -53,6 +53,6 @@ module RSpec
   module Expectations
     remove_const :ExpectationNotMetError
     # Exception raised when an expectation fails.
-    ExpectationNotMetError = ::Minitest::Assertion
+    const_set('ExpectationNotMetError', ::Minitest::Assertion)
```

Comment:
Can this use a symbol instead?  After all, we used on for `remove_const`, and a symbol won't allocate a new object like a string will.

Reply:
it can!